docs/Open Data Model/Open Data Model.md

```diff
@@ -0,0 +1,663 @@
+﻿**Open Data Model (ODM)**
+===================
+
+**Overview**
+----
+This document describes a strategy for creating an open data model (ODM) for Apache Spot (incubating) (formerly known as “Open Network Insight (ONI)”) in support of cyber security analytic use cases. It also describes the use cases for which Apache Spot (incubating) running on the Cloudera platform is uniquely capable of addressing along with the data model.
+
+**Apache Spot (incubating) Open Data Model Strategy**
+------------------------------------
+The Apache Spot (incubating) Open Data Model (ODM) strategy aims to extend Apache Spot (incubating) capabilities to support a broader set of cyber security use cases than initially supported. The primary use case initially supported by Apache Spot (incubating) includes Network Traffic Analysis for network flows (Netflow, sflow, etc.), DNS and Proxy; primarily the identification of threats through anomalous event detection using both supervised and unsupervised machine learning.
+
+In order to support a broader set of use cases, Spot must be extended to collect and analyze other common
+“event-oriented” data sources analyzed for cyber threats, including but not limited to the following log types:
+
+> Proxy
```

Comment:
I still think these > are not necessary. They make the bulleted list appear like quotes.
Simply an unordered list would do here (https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists)